docs/Console/README.md

```diff
@@ -130,24 +129,24 @@ this mode, the solution must be manually built. To do so, please follow these
 steps:
 
 1. Clone the project: `git clone https://github.com/Microsoft/dotnet-apiport`
-2. Compile as normal, either using `msbuild` or Visual Studio
-3. If in Visual Studio, set the project `ApiPort.Offline` as the startup project, or if from command line, goto `bin\[Configuration]\ApiPort.Offline` and run `ApiPort.exe` from this directory
+2. Compile using the `build.cmd` or `build.sh` script as normal
+3. If in Visual Studio, set the project `ApiPort.Offline` as the startup project, or if from command line, goto `bin\[Configuration]\ApiPort.Offline\net46` and run `ApiPort.exe` from this directory.
```

Comment:
On my local machine it's further down in a sub directory: `...\net46\win7-x64`.  There's a setting to not add the RID in output path if we want, but probably not in this PR.

Reply:
Fixed